mkosi

```diff
@@ -2640,6 +2667,10 @@ def run_build_script(args, workspace, raw):
         else:
             cmdline.append("--chdir=/root")
 
+        if args.build_dir is not None:
+            cmdline.append("--setenv=BUILDDIR=/root/build")
```

Comment:
Shouldn't this be always set, even if no build_dir? This would break `$BUILDDIR`-using scripts if one accidentally nukes `mkosi.builddir`